randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -1941,7 +1947,7 @@ Extra - asset_id: collision_camera_024
   * Layers: default
   * Slide Tunnel to Above Pulse Radar/Tunnel to Golzuna Tower
   > Tile Group (POWERBEAM) 1
-      Morph Ball
+      Can Slide
```

Comment:
This distance is too far to reach with sliding alone if i remember correctly - I would keep this as morph ball

Reply:
Without morph ball you just keep sliding until you fall or hit a wall